howdju-docs/package.json

```diff
@@ -12,7 +12,7 @@
     "@svgr/webpack": "^8.1.0",
     "eslint": "^8.49.0",
     "eslint-config-next": "^14.0.1",
-    "next": "^13",
```

Comment:
It looks like nextra is hardcoded to work with next 13 only ([link](https://github.com/shuding/nextra/blob/67252808eddbf7f46f941bf15a5b16e5833f3cd1/package.json#L53)).